tests/network/network_wrappers/test_osm_network_wrapper.py

```diff
@@ -277,3 +274,21 @@ def test_snap_nodes_to_nodes(self, _valid_graph_fixture: MultiDiGraph):
 
         # 3. Verify expectations.
         assert isinstance(_result_graph, MultiDiGraph)
+
+    @slow_test
+    def test_given_valid_base_geometry_with_polygon(
+        self, _valid_network_polygon_fixture: BaseGeometry
+    ):
+        # 1. Define test data.
+        _network_config_data = self._get_dummy_network_config_data()
+        _network_config_data.network.network_type = NetworkTypeEnum.DRIVE.config_value
+        _network_config_data.network.road_types = ""
```

Comment:
Why need this line?

Reply:
It is just about providing a working case.